src/components/probe/probing.ts

```diff
@@ -66,31 +66,28 @@ export async function probing(
     })
     return res as AxiosResponseWithExtraData
   } catch (error) {
-    let errStatus
+    let errResponseCode
     let errData
     let errHdr
 
     if (error.response) {
       // Axios doesn't always return error response
-      errStatus = error.response.status
+      errResponseCode = error.response.status
       errData = error.response.data
       errHdr = error.response.headers
     } else {
-      errStatus = 500 // TODO: how to detect timeouts?
+      errResponseCode = 500 // TODO: how to detect timeouts?
```

Comment:
For detecting timeouts error, we can use:
```
if (error.code === 'ECONNABORTED')
```

Reply:
reff: https://github.com/axios/axios/blob/v0.21.1/lib/adapters/http.js#L280

Reply:
thanks greg.. yup found this